gradle.properties

```diff
@@ -4,36 +4,38 @@
 java_version=17
 
 # Minecraft and mappings versions
-minecraft_version=1.19.4
+minecraft_version=1.20.2
 
 # Architectury
 # Check for latest at https://maven.architectury.dev/architectury-plugin/architectury-plugin.gradle.plugin
 architectury_plugin_version=3.4-SNAPSHOT
 # Check for latest at https://maven.architectury.dev/dev/architectury/loom/dev.architectury.loom.gradle.plugin
 architectury_loom_version=1.2-SNAPSHOT
-enabled_platforms=fabric,forge,quilt
+enabled_platforms=fabric
```

Comment:
Oops..?

Reply:
Did you try this on Quilt?

Reply:
Quilt was not out yet (forge neither, but I added it back). This will be fixed.

Reply:
So are we stuck waiting for Quilt to support 1.20.2?